packages/stores/types.ts

```diff
@@ -201,6 +201,7 @@ type TClientStore = {
     setP2pAdvertiserInfo: () => void;
     setPreSwitchAccount: (status?: boolean) => void;
     switchAccount: (value?: string) => Promise<void>;
+    setLoginInformation: (client_accounts: object, client_id: string) => void;
```

Comment:
```suggestion
    setLoginInformation: (client_accounts: { [k: string]: TActiveAccount }, client_id: string) => void;
```
@yashim-deriv Not 100% sure about this one 👀